src/panels/config/backup/ha-config-backup-settings.ts

```diff
@@ -196,10 +159,10 @@ class HaConfigBackupSettings extends LitElement {
 
   private _encryptionKeyChanged(ev) {
     const password = ev.detail.value as string;
-    this._backupConfig = {
-      ...this._backupConfig,
+    this.config = {
```

Comment:
I don't think we should change a public property, it could be overwritten by the parent before it is saved.

Reply:
It was a state before, I forgot to change this logic